exchanges/huobi/huobi_types.go

```diff
@@ -1,5 +1,2431 @@
+}
+
+// SwapTradingFeeData stores trading fee data for swaps
+type SwapTradingFeeData struct {
+	Data []struct {
+		Symbol        string  `json:"symbol"`
+		ContractCode  string  `json:"contract_code"`
+		FeeAsset      string  `json:"fee_asset"`
+		OpenMakerFee  float64 `json:"open_maker_fee,string"`
+		OpenTakerFee  float64 `json:"open_taker_fee,string"`
+		CloseMakerFee float64 `json:"close_maker_fee,string"`
+		CloseTakerFee float64 `json:"close_taker_fee,string"`
+	} `json:"data"`
+	Timestamp int64 `json:"ts"`
+}
+
+// TransferLimitData stores transfer limits
+type TransferLimitData struct {
+	Data []struct {
+		Symbol                 string  `json:"symbol"`
+		ContractCode           string  `json:"contract_code"`
+		MaxTransferIn          float64 `json:"transfer_in_max_each"`
+		MinTransferIn          float64 `json:"transfer_in_min_each"`
+		MaxTransferOut         float64 `json:"transfer_out_max_each"`
+		MinTransferOut         float64 `json:"transfer_out_min_each"`
+		MaxTransferInDaily     float64 `json:"transfer_in_max_daily"`
+		MinTransferInDaily     float64 `json:"transfer_in_min_daily"`
+		MaxTransferOutDaily    float64 `json:"transfer_out_max_daily"`
+		MinTransferOutDaily    float64 `json:"transfer_out_min_daily"`
+		NetTransferInMaxDaily  float64 `json:"net_transfer_in_max_daily"`
+		NetTransferOutMaxDaily float64 `json:"net_transfer_out_max_daily"`
+	} `json:"data"`
+	Timestamp int64 `json:"timestamp"`
+}
+
+// PositionLimitData stores position limit data
+type PositionLimitData struct {
+	Data []struct {
+		Symbol       string  `json:"symbol"`
+		ContractCode string  `json:"contract_code"`
+		BuyLimit     float64 `json:"buy_limit"`
+		SellLimit    float64 `json:"sell_limit"`
+	} `json:"data"`
+	Timestamp int64 `json:"ts"`
+}
+
+// InternalAccountTransferData stores transfer data between subaccounts and main account
+type InternalAccountTransferData struct {
+	TS   int64 `json:"ts"`
+	Data struct {
+		OrderID string `json:"order_id"`
+	} `json:"data"`
+}
+
+// InternalAccountTransferRecords stores data for transfer records within the account
+type InternalAccountTransferRecords struct {
+	Timestamp string `json:"ts"`
+	Data      struct {
+		TransferRecord []struct {
+			ID             int64   `json:"id"`
+			Timestamp      int64   `json:"ts"`
+			Symbol         string  `json:"symbol"`
+			SubUID         int64   `json:"sub_uid"`
+			SubAccountName string  `json:"sub_account_name"`
+			TransferType   int64   `json:"transfer_type"`
+			Amount         float64 `json:"amount"`
+		} `json:"transfer_record"`
+		TotalPage   int64 `json:"total_page"`
+		CurrentPage int64 `json:"current_page"`
+		TotalSize   int64 `json:"total_size"`
+	} `json:"data"`
+}
+
+// SwapOrderData stores swap order data
+type SwapOrderData struct {
+	Data struct {
+		OrderID       int64  `json:"order_id"`
+		OrderIDString string `json:"order_id_string"`
+		ClientOrderID int64  `json:"client_order_id"`
+	} `json:"data"`
+	Timestamp string `json:"ts"`
```

Comment:
I'm not seeing any instance of `ts` being a string for any huobi endpoints, are these tested? https://huobiapi.github.io/docs/coin_margined_swap/v1/en/#order-details-acquisition